docs/variants/msi_z690/cpu-hcl.md

```diff
@@ -2,41 +2,158 @@
 
 ## Contributing
 
-Feel free to test different processors and report your results via
-[email](mailto:contact@dasharo.com) or submit a Pull Request to
-[Dasharo documentation repository](https://github.com/Dasharo/docs) or by using
-[Dasharo issues repository](https://github.com/Dasharo/dasharo-issues/issues).
+* Use [Dasharo Tools Suite HCL
```

Comment:
Maybe we need to rename the file, if it contains more than just CPU HCL?

Reply:
@macpijan fixed: https://github.com/Dasharo/docs/pull/463/commits/b15568a7f22156f1106be853b27ad78feecbcd94